File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusOrganizationForm.kt
package com.sdkwork.backend

data class PlusOrganizationForm(
    val name: String? = null,
    val code: String? = null,
    val status: String? = null,
    val description: String? = null,
    val parentId: Int? = null,
    val installAppList: InstallAppList? = null
)

File: sdkwork-backend-sdk-csharp/Models/PlusOrganizationForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusOrganizationForm
    {
        public string? Name { get; set; }
        public string? Code { get; set; }
        public string? Status { get; set; }
        public string? Description { get; set; }
        public int? ParentId { get; set; }
        public InstallAppList? InstallAppList { get; set; }
    }
}
